services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -61,207 +111,52 @@ public ContentMigrationAction(MigrationSourceContext sourceContext, IApiService
     public override async Task PerformActionAsync<T>(IIngestServiceActionManager manager, string? name = null, T? data = null, CancellationToken cancellationToken = default) where T : class
     {
         this.Logger.LogDebug("Performing ingestion service action for ingest '{name}'", manager.Ingest.Name);
+        IContentMigrator contentMigrator = _migratorFactory.GetContentMigrator(manager.Ingest.Name);
 
-        // var lookups = await this.Api.GetLookupsAsync();
+        var lookups = await this.Api.GetLookupsAsync();
+        IEnumerable<IngestModel> ingests = await this.Api.GetIngestsAsync();
 
         var skip = 0;
-        var count = 20;
-        DateTime? importDateStart = manager.Ingest.GetConfigurationValue<DateTime>("importDateStart");
-        DateTime? importDateEnd = manager.Ingest.GetConfigurationValue<DateTime>("importDateEnd");
-        DateTime? creationDateOfNewsItem = manager.Ingest.GetConfigurationValue<DateTime>("creationDateOfLastImport");
+        var count = 50;
```

Comment:
Value like this would be handle to be part of the configuration options so that it can be controlled without editing code.